Makefile

```diff
@@ -98,7 +98,8 @@ test_unit: clean static_external
 
 # Check whether migrations need to be generated
 test_migrations_missing: clean
-	! honcho -e .env.test run ./manage.py makemigrations --dry-run --exit
+	(psql -lqt | cut -d \| -f 1 | grep -qw opencraft) || (createdb --encoding utf-8 --template template0 opencraft)
```

Comment:
If I understand correctly, this new line creates the DB if it doesn't exist, right? Why is this needed here?